Fix #1007 - Call save-device-info method at application launch

cagryInside opened the pull request:
Calling the `saveDeviceInfo` method in the app launch.  Previously, we were calling this method on every transition event and it was causing duplicate ~device-information` records on Firebase.  Now, we only call once when the app is started.  The new data looks like this on Firebase:

![Screen Shot 2019-07-18 at 3 21 10 PM](https://user-images.githubusercontent.com/2777974/61496002-b2ef4500-a96f-11e9-9993-88734df04884.png)

barbeau commented:
Thanks @cagryInside! How about putting the timestamp as a field too so we can query on it?

cagryInside commented:
> Thanks @cagryInside! How about putting the timestamp as a field too so we can query on it?

@barbeau good idea.  Just added and updated the commit:

![Screen Shot 2019-07-19 at 11 08 12 AM](https://user-images.githubusercontent.com/2777974/61556173-0e740e00-aa16-11e9-840c-4ad1fde3340a.png)

barbeau commented:
Thanks @cagryInside!
